untitled/src/RobotMove.java

```diff
@@ -0,0 +1,49 @@
+class RobotMove {
+
+		public enum Direction {
```

Comment:
better move enum into separate file

Reply:
Fix all from comments and test all on stepik